com.smallstep.Smallstep.yml

```diff
@@ -0,0 +1,101 @@
+id: com.smallstep.Smallstep
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: smallstep
```

Comment:
I removed the wrapper. The desktop/gui binary starts without any arguments on all platforms.